src/main/java/com/epam/ta/reportportal/core/launch/impl/UpdateLaunchHandlerImpl.java

```diff
@@ -162,7 +163,11 @@ public OperationCompletionRS startLaunchAnalyzer(ReportPortalUser.ProjectDetails
 		List<Long> itemIds = collectItemsByModes(project, user.getUsername(), launch.getId(), analyzeRQ.getAnalyzeItemsMode());
 
 		analyzerServiceAsync.analyze(launch, itemIds, analyzerConfig)
-				.thenApply(it -> logIndexer.indexItemsLogs(project.getId(), launch.getId(), itemIds, analyzerConfig));
+				.thenApply(it -> CompletableFuture.supplyAsync(() -> logIndexer.indexItemsLogs(project.getId(),
```

Comment:
i think we don't need indexing to be async here, cause it is already async for analyze

Reply:
fixed